exs24.lua

```diff
@@ -32,7 +32,7 @@ function load_exs(path)
 
   while (i + 84 < data_size) do
     fh:seek("set", i)
-    local sig = read_dword(fh, big_endian)
+    local sig = read_dword(fh, false)
```

Comment:
I'm not sure why but it does look like the 'sig' should always be little endian 👍. [This library does the same thing](https://github.com/ibireme/yy_music_base/blob/9936b096623dc1e018d342d06ac68344ab5c395e/yy_music_base/ym_exs.c#L550).